### DISCUSSION TITLE
Argos Translate for Mac direct purchase

### DISCUSSION THREAD

**Discussion by argosopentech** [argosopentech/argos-translate#131] [General]
The Mac app is ready to go but Apple is taking forever with approval so we’re going to start doing direct sales. To purchase please email ([admin@argosopentech.com](admin@argosopentech.com)) with payment information and get access to Argos Translate packaged for macOS!

## $20
- [PayPal](paypal.me/argosopentech)
- Ethereum: 0x4E9017d8e275cA54C91E056381DAb9fe6ECC1AF6
- Bitcoin cash: qzqklgjpgutdqqlhcasmdd2hkqcelw426sxzk5qtne
- Bitcoin: 16UJrmSEGojFPaqjTGpuSMNhNRSsnspFJT
- Filecoin: f1nrnpmjxn27amidyiqrzq5mxihdo2trh2oijw2sq
- Basic Attention Token: 0x8a16f26D277f924B04FCA5ECec64b76B5410A06c
- Cheap Eth: 0x996133E61b81c300a37ACa9b24898685eB872b61

## Supported Languages
- English
- Spanish
- German
- Russian
- French
- Portuguese

## Mac App
The macOS native version uses the new [seq2seq sentence boundary detection system](https://www.youtube.com/watch?v=TyFRbg7rsuE&list=PLe6dpCSdH0zSXUJiZhWfwYzoIWmvCzdhH) and is able to run without Stanza.

![Argos Translate MacOS](https://user-images.githubusercontent.com/48267258/121823401-b9c18800-cc6a-11eb-94d5-4110d58ed4de.png)

**Comment by ww7** [comment]
Are you considering monetization through Setapp?

**Reply by PJ-Finlay** [reply]
Thanks for the suggestion! I tried contacting them but haven't heard back. It looks promising though.

**Comment by PJ-Finlay** [comment]
I've given up on publishing to the Apple Store for the time being but the app runs fine is is still available for direct purchase.

**Comment by PJ-Finlay** [comment]
Alright no one's bought this, you can have it for free if you want! Donations still appreciated though.

https://www.fulfillment.argosopentech.com/argos-translate/macos/1981867401692101689462823333574892722487703316879849724390796876302268929269575834660004918158927127779617803719686848953512695878948840276478203680869921/

**Reply by PJ-Finlay** [reply]
This is now distributed via GitHub releases
